### PR TITLE
Add Docker support

### PR DESCRIPTION
**Pull request by melkamar**
This MR adds support for running the script in Docker, so that there is no hassle with installing dependencies etc.

I am using an image built from my fork of the repository, but feel free to set up Docker Hub integration and update the README to point to an image built from the original repository.

https://hub.docker.com/r/melkamar/geo-heatmap

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,6 +107,19 @@ optional arguments:
...
 
+## Running with Docker
+
+With Docker this script can be executed without the need for installing any dependencies. 
+
+Assume that the Google takeout archive is located here: `/some/path/takeout.zip`. Run the following
+Docker command to generate a `heatmap.html` file under `/some/path/`:
+
+```bash
+$ docker run --rm -v /some/path/:/data melkamar/geo-heatmapa /data/takeout.zip
```
Comment (SuperSandro2000) [inline README.md]: ``melkamar`` is probably not the desired username of the Docker image.

Also i would be better to suggest building the Dockerimage locally unless luka sets up the building himself. We handle quite personally data here and allowing your image to be used is a major attack vector and shouldn't be trusted.

**Comment by luka1199** [comment]
I don't know much about docker but I will maybe look into it in the future. If it could be a security issue to not use the original repo I can't merge this.

File: README.md (hunk 1 above)
Comment (melkamar) [reply README.md]: Yeah, as mentioned in the comment, the idea is that @luka1199 can create their own image in Docker Hub and then change the name of the image. I cannot publish the Docker image in their namespace.

I wanted to make it as quick as possible to use, but building the image locally could be a reasonable compromise between speed and trust. I'll add that to the readme, thanks.

**Comment by melkamar** [comment]
Fair enough, I'll remove the Docker Hub image reference and just add steps to build this locally. That would not pose any security issue, as all that does is use the files in this repository locally, not uploading nor downloading anything anywhere. Would that be acceptable?

This is mainly a simplification for people already using Docker. They would not have to install any local Python dependencies and clutter their system. For people going this way, there would also (probably) not be any Python-related issue reports that were caused by incorrect packages or other environment misconfiguration.

**Comment by SuperSandro2000** [comment]
Building the image themselves works for me. No real issue there.

**Comment by luka1199** [comment]
Hey the pull request got closed because I removed some files from the history. Can you rebase on the new fresh history and submit another pull request? Sorry for the inconvenience 😅

File: README.md (hunk 1 above)
Comment (jamesjarvis) [reply README.md]: Suggestion to use github package repository https://help.github.com/en/github/managing-packages-with-github-packages/about-github-packages#supported-clients-and-formats to host the docker image, therefore there wouldn't be a namespace issue as it is per repository

**Pull request closed without merging**